.github/workflows/build.yml

```diff
@@ -0,0 +1,59 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request_target:
```

Comment:
If you add `workflow_dispatch`, we can start the action manually.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch